### ISSUE TITLE
change metric to delegates?

### ISSUE THREAD

**Issue by mcwhittemore** [is-trump-still-winning/is-trump-still-winning.github.io#26]
Now that we are in a real race, it would be great to use a real metric, state delegates.

**Comment by mcwhittemore** [comment]
https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&es_th=1&ie=UTF-8#q=delegate%20count%20republican%20primary&eob=m.09c7w0/R/2/full/m.09c7w0/&es_th=1

**Comment by conorhastings** [comment]
agreed, also found this one http://www.realclearpolitics.com/epolls/2016/president/republican_delegate_count.html
